tools/topology/topology2/platform/intel/deep-buffer.conf

```diff
@@ -29,6 +29,11 @@ Object.PCM.pcm [
 		Object.PCM.pcm_caps.1 {
 			name $DEEP_BUFFER_PCM_NAME
 			formats 'S16_LE,S24_LE,S32_LE'
+			# align with $DEEPBUFFER_FW_DMA_MS
+			period_size_min		768
+			period_size_max		65535
+			buffer_size_min		131072
+			buffer_size_max		262144
```

Comment:
how does this allow for 32768? it doesnt seem to be in the buffer_size range?

Reply:
Think PCM capabilities uses size_t and --period-size uses snd_pcm_uframes_t.
Maybe it's a better idea to keep period_size_min/buffer_size_min unchanged so that we can use for ref comparison purpose.